sphinxcadquery/sphinxcadquery.py

```diff
@@ -159,14 +159,14 @@ def copy_asset_files(app, exc):
 def setup(app):
     setup.app = app
     app.connect('build-finished', copy_asset_files)
-    app.add_javascript('sphinxcadquerystatic/three.js')
-    app.add_javascript('sphinxcadquerystatic/AMFLoader.js')
-    app.add_javascript('sphinxcadquerystatic/STLLoader.js')
-    app.add_javascript('sphinxcadquerystatic/LegacyJSONLoader.js')
-    app.add_javascript('sphinxcadquerystatic/jszip.min.js')
-    app.add_javascript('sphinxcadquerystatic/OrbitControls.js')
-    app.add_javascript('sphinxcadquerystatic/WebGL.js')
-    app.add_javascript('sphinxcadquerystatic/main.js')
+    app.add_js_file('sphinxcadquerystatic/three.js')
+    app.add_js_file('sphinxcadquerystatic/AMFLoader.js')
+    app.add_js_file('sphinxcadquerystatic/STLLoader.js')
+    app.add_js_file('sphinxcadquerystatic/LegacyJSONLoader.js')
+    app.add_js_file('sphinxcadquerystatic/jszip.min.js')
+    app.add_js_file('sphinxcadquerystatic/OrbitControls.js')
+    app.add_js_file('sphinxcadquerystatic/WebGL.js')
+    app.add_js_file('sphinxcadquerystatic/main.js')
     app.add_stylesheet('sphinxcadquerystatic/main.css')
```

Comment:
I suspect `add_stylesheet` will also throw an error. Can you test everything is working as expected locally before we merge this? :blush:

Reply:
Thanks for the hints on the issue, this builds docs locally now. You are right it also required add_stylesheet -> add_css_file .